exporter/kafkaexporter/config.go

```diff
@@ -42,6 +42,9 @@ type Config struct {
 	// Client, and shared by the Producer/Consumer.
 	Metadata Metadata `mapstructure:"metadata"`
 
+	// Producer is the namespaces for producer properties used only by the Producer
+	Producer Producer `mapstructure:"producer"`
```

Comment:
I really don't like this... Half of the config follows sarama config the other half follows otel config style....

Reply:
Not your fault please open an issue to re-evaluate the config for this.